entrypoint.sh

```diff
@@ -42,6 +43,13 @@ elif [ -z "$(git diff-index HEAD)" ]; then
   echo "COMMAND did not change the status of the repository. Exiting."
   exit 1
 else
+
+# Set TARGET branch
+if [[ -z "${INPUT_TARGET}" ]]; then
+  TARGET="master"
+else
+  TARGET="main"
```

Comment:
Now, the value of INPUT_TARGET is used as a bool, which is confusing considering the name of the variable. It would be more clear with
```
  if [[ -z "${INPUT_TARGET}" ]]; then
    TARGET="master"
  else
    TARGET="${INPUT_TARGET}"
```
 or
```
  if [[ -z "${USE_MAIN_BRANCH}" ]]; then
    TARGET="master"
  else
    TARGET="main"
```